builder/make.sh

```diff
@@ -7,3 +7,25 @@
 
```

Comment:
Make sure to not execute other commands on failure:
```suggestion
set -e
```